Integrations/integration-Cloaken.yml

```diff
@@ -0,0 +1,112 @@
+description: Unshorten urls through a tor proxy server. Unshorten urls onsite with
+  the power of Tor to prevent leaking ip addresses to adversaries.
+detaileddescription: |-
+  Cloaken will return the unshortened version of a url.
+  Created by CypherInt.  Visit us at https://cypherint.com.  Subscribe on the Amazon marketplace: https://aws.amazon.com/marketplace/
+configuration:
+- display: Username
+  name: credentials
+  defaultvalue: ""
+  type: 9
+  required: true
+- display: Server URL (e.g. https://cloaken.cypherint.com)
+  name: server_url
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: Proxy Url
+  name: proxy
+  defaultvalue: ""
+  type: 0
```

Comment:
This should be boolean parameter, and the URL is taken from the server configuration set on Demisto.
Please use `handle_proxy()` function to handle the proxy properly in the code.

Reply:
done